symfony/monolog-bundle/3.1/config/packages/test/monolog.yaml

```diff
@@ -0,0 +1,7 @@
+monolog:
+    handlers:
+        main:
+            type: stream
+            path: "%kernel.logs_dir%/%kernel.environment%.log"
```

Comment:
Question: Why don't we hardocde the "test.log" filename ? (same in other env)

Reply:
I cannot come up with a good reason. The result would be the same...

Reply:
Yes, indeed ;)